File: api/HotDeskApplicationApi/ModelView/ReservationView.cs
using Microsoft.EntityFrameworkCore;

namespace HotDeskApplicationApi.NewFolder2
{
    [Keyless]
    public class RegistrationView
    {
        public Guid ReservationID { get; set; }

        public DateTime ArrivalTime { get; set; }

        public DateTime LeavingTime { get; set; }

        public string? OfficaName { get; set; }

        public string? FloorName { get; set; }

        public string? DeskName { get; set; }

        public byte[]? Avatar { get; set; }

        public string? ProfileRole { get; set; }

        public string? ProfileName { get; set; }
    }
}

File: web/src/models/reservationView.ts
export interface ReservationView {
    reservationID : string;
    arrivalTime :Date ;
    leavingTime :Date;
    officaName : string;
    floorName : string;
    deskName :string;
    avatar: Uint8Array;
    profileRole : string;
    profileName : string;
  }